### DISCUSSION TITLE
Add a view_table in form as a EntityType class

### DISCUSSION THREAD

**Discussion by heardofdezz** [symfony/symfony#43503] [Q&A]
I'm facing a tough dilemma I have a view_table and I'm trying to include that view_table into my form as a EntityType class, without creating the view_table Class.

The goal is to avoid modifying my database schema, most of the answered I saw so far were pushing for creating a table into my schema something I'm trying to avoid.

I there anyway I can be able to call my view_table without going through such process.

**Comment by ghost** [comment]
To be honest, I'm not sure I understand the problem but I'll try to help you until a more competent developer comes to your help. 

I understand that you are trying to exploit the data of a table in your db while keeping it independent from the Doctrine mapping system.

If this is the case, are you sure you can't do an SQL query (but I'm not sure you can [query tables not mapped with Doctrine](https://www.doctrine-project.org/projects/doctrine-orm/en/2.9/reference/native-sql.html)) and customize a ChoiceType field in order to exploit your data the way you want? 
Indeed, the EntityType field seems to be a special _[ChoiceType field that's designed to load options from a Doctrine entity](https://symfony.com/doc/current/reference/forms/types/entity.html)_.

The Form component provides many tools such as transformers and events to interact with your data in detail :) 

Anyway, good luck!

**Reply by mbabker** [reply]
With the DBAL, you can query any table.  With the ORM, you have to have an object (entity) to work with.

So, the `EntityType` field 100% requires entity classes (or, for the ODM users reading this, a document class).
